consultation_analyser/consultations/models.py

```diff
@@ -86,17 +86,28 @@ class Meta(UUIDPrimaryKeyModel.Meta, TimeStampedModel.Meta):
 
 
 class Theme(UUIDPrimaryKeyModel, TimeStampedModel):
+    # Label summarises a topic from a topic model for a given question
     label = models.CharField(max_length=256, blank=True)
```

Comment:
Do we need to hold on to label if we have keywords?

Reply:
The label allows us to calculate the `is_outlier` and the `keywords`. I think `keywords` are unique up to label so maybe we don't need the label? Not sure of the cleanest way to write this without the `label` field - open to suggestions.

Reply:
Let's keep for now. We can always remove later